src/presets.asm

```diff
@@ -297,7 +294,6 @@ preset_start_gameplay:
     JSL $80835D  ; Disable NMI
     JSL $80985F  ; Disable horizontal and vertical timer interrupts
     JSL $82E76B  ; Load destination room CRE bitset, door/room/state headers, tiles
-    JSR $A12B    ; Play 14h frames of music
```

Comment:
I thought I tried removing this once and had problems, but maybe I made some other changes at the same time that were no good, or maybe the music handling is just better now.  I didn't have problems removing this.